Undefined name: config in separate_api_route_example.py

## Context

$ `flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics`
```
./aiogram/examples/separate_api_route_example.py:10:17: F821 undefined name 'config'
bot = Bot(token=config.bot_token)
                ^ 
```
The [other instances of `Bot(token`](https://github.com/aiogram/aiogram/search?q=Bot%28token) use a hard coded TOKEN constant.
This example gets the token from `config` but that is not imported or defined.

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Operating System:
* Python Version: 
* aiogram version: 
* aiohttp version: 
* uvloop version (if installed): 

## Expected Behavior

Please describe the behavior you are expecting

## Current Behavior

What is the current behavior?

## Failure Information (for bugs)

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. step 1
2. step 2
3. you get it...

### Failure Logs

Please include any relevant log snippets or files here.

--- a/examples/separate_api_route_example.py
+++ b/examples/separate_api_route_example.py
@@ -7,7 +7,9 @@ from aiogram.dispatcher.webhook import configure_app
 from aiohttp import web
 
 
-bot = Bot(token=config.bot_token)
+API_TOKEN = "BOT_TOKEN_HERE"
+
+bot = Bot(token=API_TOKEN)
 dp = Dispatcher(bot)
 
 

